bedrock/settings/base.py

```diff
@@ -366,11 +366,20 @@ def set_whitenoise_headers(headers, path, url):
     }
 }
 
+
+def get_app_name(hostname):
+    if hostname.startswith('bedrock-'):
+        app_mode = hostname.split('-')[1]
+        return 'bedrock-' + app_mode
+
+    return None
+
+
 HOSTNAME = platform.node()
-DEIS_APP = config('DEIS_APP', default='')
-DEIS_DOMAIN = config('DEIS_DOMAIN', default='')
+APP_NAME = get_app_name(HOSTNAME)
+CLUSTER_NAME = config('CLUSTER_NAME', default='')
```

Comment:
Is there anything we need to do to ensure this is set on the clusters prior to merging/deploying this PR?

Reply:
I took [this line in the configmap](https://github.com/mozmeao/www-config/blob/git-sync-operator/iowa-a/bedrock-prod/configmap.yaml#L10) to mean that it's ready. @jgmize can you confirm that it's ready and available in every cluster including oregon?

Reply:
@jgmize actually I don't see `CLUSTER_NAME` in oregon deployments. I'll set that via deis commands and then merge.